package.json

```diff
@@ -5,12 +5,12 @@
     "main": "index.js",
     "scripts": {
         "dev": "nodemon",
-        "dev-server": "cross-env NODE_ENV=development babel-node ./src/server/",
+        "dev-server": "cross-env NODE_ENV=development babel-node -r './src/server/tracing.js' ./src/server/",
         "build-frontend": "next build",
         "build-server": "babel ./src/i18n.js ./src/constants.js --out-dir dist/ && babel ./src/server --out-dir dist/server && babel ./src/common --out-dir dist/common",
         "build-docs": "jsdoc -r -d generated-docs ./src",
         "build": "run-p build-frontend build-server build-docs",
-        "start": "cross-env NODE_ENV=production node dist/server/index.js",
+        "start": "cross-env NODE_ENV=production node -r './dist/server/tracing.js' dist/server/index.js",
```

Comment:
I was wondering why the tracing code to register the SDK couldn't be called inside `server/index.js`, but the couple of opentelemetry docs I came across both recommend that this tracing code should be loaded before the app loads 🤷

Reply:
It has to do with the automatic instrumentation, I think -- it needs to be loaded before the module it instruments is loaded, otherwise it won't be able to override the behavior to add the instrumentation. I think that it can sorta work if you can put it right at the top of the first thing you load... but this is much clearer (since it's more explicit about the intent) and probably less brittle. So I figured this was the way to go.

Reply:
Sounds good to me 👍